src/app/Random.js

```diff
@@ -0,0 +1,41 @@
+const recipe = document.querySelector("#recipiesRandom");
+
+function getRandomRecipes() {
+    recipe.replaceChildren();
+
+    fetch('https://api.spoonacular.com/recipes/random?apiKey=fa79327224724e9da0b733fdcc9720d4')
+        .then(response => response.json())
+        .then(data => {
+            const img = document.createElement("img");
+            img.setAttribute("src", data.recipes[0].image);
+            recipe.appendChild(img);
+
+            const ingeredients = document.createElement("div");
+
+            function ingeredientsName() {
+                let ingredientsArray = [];
+                for (i = 0; i < data.recipes[0].extendedIngredients.length; i++) {
+
+                    ingredientsArray.push(data.recipes[0].extendedIngredients[i].name);
+
+                }
+                return ingredientsArray;
+            }
+
+            ingeredients.innerHTML = ` <h4>Ingerdients:</h4> ${ingeredientsName()}`;
+            recipe.appendChild(ingeredients);
+
+            const instructions = document.createElement("div");
+            instructions.innerHTML = ` <h3>Instruction:</h3> ${data.recipes[0].instructions}`;
+
+            recipe.appendChild(instructions);
+
+        });
+}
+function getRandomRecipesInterval() {
+    setInterval(getRandomRecipes, 10000);
+} 
+
+
+document.getElementById("random").onclick = getRandomRecipes; 
+document.getElementById("randomInterval").onclick = getRandomRecipesInterval; 
```

Comment:
Please research which solution is better

Reply:
addEventListener won't work in IE versions less than 9, on the other hand onclick has this drawback "The significant drawback with inline events is that unlike event listeners described above, you may only have one inline event assigned. Inline events are stored as an attribute/property of the element[doc], meaning that it can be overwritten."
Also it's worth to note that modern frameworks like Angular transpile their onclick functions to addEventLister, full comparison here, so it's likely that addEventListener will have better support in the future
Source and full comparison:
https://stackoverflow.com/questions/6348494/addeventlistener-vs-onclick